modules/chain-grabber/src/main/scala/org/ergoplatform/explorer/indexer/models/HeightsResult.scala

```diff
@@ -0,0 +1,28 @@
+package org.ergoplatform.explorer.indexer.models
+
+import cats.Applicative
+import org.ergoplatform.explorer.Err.ProcessingErr.IncorrectNetworkView
+import org.ergoplatform.explorer.Err.RequestProcessingErr.InconsistentDbData
+import org.ergoplatform.explorer.{CRaise, Id, UrlString}
+import tofu.Raise
+import tofu.logging.Loggable
+import tofu.syntax.monadic._
+import tofu.syntax.raise._
+
+import scala.collection.immutable.SortedMap
+
+final case class HeightsResult(
+  bestHeight: Int,
+  distinguishHeights: List[(UrlString, Int)]
+)
+
+object HeightsResult {
+
+  def apply[F[_]: CRaise[*[_], IncorrectNetworkView]: Applicative](heights: List[(UrlString, Int)]): F[HeightsResult] = {
```

Comment:
Also, why passing an array to method where you just throw everything except the URL with max index?